subworkflows/local/download_references/main.nf

```diff
@@ -62,11 +65,30 @@ workflow DOWNLOAD_REFERENCES {
         ch_built_vep_plugin_file = Channel.empty()
     }
 
+    if ( download_drop_mae_high_q_vcf ) {
+        if (ch_genome.contains('38')) {
+            ch_input_hg38 = Channel.of( tuple('qc_vcf_1000G_hg38.vcf.gz', 'https://www.cmm.in.tum.de/public/paper/drop_analysis/resource/qc_vcf_1000G_hg38.vcf.gz') )
+            ch_input_hg38_tbi = Channel.of( tuple('qc_vcf_1000G_hg38.vcf.gz.tbi', 'https://www.cmm.in.tum.de/public/paper/drop_analysis/resource/qc_vcf_1000G_hg38.vcf.gz.tbi') )
+            WGET_VCF_MAE(ch_input_hg38)
+            WGET_VCF_MAE_TBI(ch_input_hg38_tbi)
+        } else {
+            ch_input_hg19 = Channel.of( tuple('qc_vcf_1000G_hg19.vcf.gz', 'https://www.cmm.in.tum.de/public/paper/drop_analysis/resource/qc_vcf_1000G_hg19.vcf.gz') )
+            ch_input_hg19_tbi = Channel.of( tuple('qc_vcf_1000G_hg19.vcf.gz.tbi', 'https://www.cmm.in.tum.de/public/paper/drop_analysis/resource/qc_vcf_1000G_hg19.vcf.gz.tbi') )
+            WGET_VCF_MAE(ch_input_hg19)
+            WGET_VCF_MAE_TBI(ch_input_hg19_tbi)
+        }
+
+        ch_high_q_vcf_tbi = WGET_VCF_MAE.out.downloaded_file.concat(WGET_VCF_MAE_TBI.out.downloaded_file)
+    } else {
+        ch_high_q_vcf_tbi = Channel.empty()
```

Comment:
Maybe this subworkflow would be cleaner if `ch_high_q_vcf_tbi` was initiated in the beginning of the script like how ch_versions is done. Then we could remove the last `else` statement. 

This could probably be done for all the "final channels" from the download if blocks.